src/main/scala/codacy/checkstyle/CodacyListener.scala

```diff
@@ -26,7 +26,7 @@ class CodacyListener extends AuditListener {
           Source.Line(event.getLine)
         )
       }
-  }
+  } catch { case NonFatal(e) => e.printStackTrace() }
```

Comment:
not sure is a good idea to print to std-out, maybe we can generate a tool failure with a proper error message

Reply:
Refactored non handling the exception at all. So if there are problems we are aware of it.